components/p3a/metric_names.h

```diff
@@ -284,6 +284,15 @@ inline constexpr auto kEphemeralHistograms =
     "Brave.WebTorrent.UsageWeekly"
 });
 
+// List of metrics which will include the stored refcode when transmitted
```

Comment:
perhaps we can start thinking about introducing an abstraction of self-describing metric? Or different methods to emit a metric, instead of maintaining a bunch of lists. That would require a custom P3A API for content, components and above